app/models/work.rb

```diff
@@ -595,11 +594,7 @@ def s3_object_persisted?(s3_file)
     end
 
     def publish_precurated_files(user)
-      # An error is raised if there are no files to be moved
```

Comment:
This is a private method that only gets called if the work is valid to approve.  That already makes sure there are files, so why check this again?  I could not write a test to test this code so I removed it.